### PR TITLE
chore: deprecate quadatric slippage and stop using it.

### LINKED ISSUE #9934: Remove quaddratic slippage
https://github.com/vegaprotocol/specs/pull/2011

### PR DESCRIPTION
**Pull request by jeremyletang**
close #9934

### CODE REVIEW

File: datanode/gateway/graphql/schema.graphql [hunk 1]
```diff
@@ -4298,7 +4298,7 @@ type UpdateMarketConfiguration {
   "Linear slippage factor is used to cap the slippage component of maintenance margin - it is applied to the slippage volume."
   linearSlippageFactor: String!
   "Quadratic slippage factor is used to cap the slippage component of maintenance margin - it is applied to the square of the slippage volume."
-  quadraticSlippageFactor: String!
+  quadraticSlippageFactor: String! @deprecated(reason: "This field will be removed in future release")
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:4301]:
```suggestion
  quadraticSlippageFactor: String! @deprecated(reason: "This field will be removed in a future release")
```

File: datanode/gateway/graphql/schema.graphql [hunk 2]
```diff
@@ -4261,7 +4261,7 @@ type NewMarket {
   "Linear slippage factor is used to cap the slippage component of maintenance margin - it is applied to the slippage volume"
   linearSlippageFactor: String!
   "Quadratic slippage factor is used to cap the slippage component of maintenance margin - it is applied to the square of the slippage volume"
-  quadraticSlippageFactor: String!
+  quadraticSlippageFactor: String! @deprecated(reason: "This field will be removed in future release")
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:4264]:
```suggestion
  quadraticSlippageFactor: String! @deprecated(reason: "This field will be removed in a future release")
```

File: datanode/gateway/graphql/schema.graphql [hunk 3]
```diff
@@ -2313,7 +2313,7 @@ type Market {
   linearSlippageFactor: String!
 
   "Quadratic slippage factor is used to cap the slippage component of maintainence margin - it is applied to the square of the slippage volume"
-  quadraticSlippageFactor: String!
+  quadraticSlippageFactor: String! @deprecated(reason: "This field will be removed in future release")
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:2316]:
```suggestion
  quadraticSlippageFactor: String! @deprecated(reason: "This field will be removed in a future release")
```

**Review by candida-d — COMMENTED** [review]
very small change

**Review by ValentinTrinque — APPROVED** [review]

**Review by peterbarrow — APPROVED** [review]

**Review by ValentinTrinque — APPROVED** [review]

**Review by EVODelavega — APPROVED** [review]

**Pull request merged**